### DISCUSSION TITLE
Conan as a package ecosystem for dependencies

### DISCUSSION THREAD

**Discussion by Kicer86** [community/community#26383] [Actions]
<p>Hi</p>
<p>I wonder why isn’t conan available as a supported package ecosystem (<a href="https://docs.github.com/en/free-pro-team@latest/github/visualizing-repository-data-with-graphs/about-the-dependency-graph#supported-package-ecosystems" rel="noopener nofollow ugc">Supported package ecosystems</a>) for c++?</p>
<p>Are there plans to add it and if not then why?</p>

**Comment by whitneyimura** [comment, marked as answer]
<p>Hi <a class="mention" href="/kicer86">@Kicer86</a> – We have it on our radar, but unfortunately do not have a date to share yet.  We’ll update the roadmap when we have landed on a date: <a href="https://github.com/github/roadmap" rel="noopener nofollow ugc">https://github.com/github/roadmap</a></p>

**Comment by robindegen** [comment]
Any update on this, since this was anwered 3 years ago? It surprised me that the largely advertised Github Packages does not have support for something as basic as Conan. Apparently even vcpkg isn't support; and that is a Microsoft product.

**Comment by p0fi** [comment]
I would also like to know the status of this :)
